fastestimator/util/latex_util.py

```diff
@@ -129,3 +129,53 @@ def __init__(self,
             data.append(bold(escape_latex(name)) if bold_name else escape_latex(name))
         data.append(NoEscape("}}}"))
         super().__init__(data=data)
+
+
+class IterJoin(Container):
+    """A class to convert an iterable to a latex representation.
+
+    The data of this class can be any type. Usually it is iterable due to its capability to setup interval string.
```

Comment:
'set up'. But also, what will happen if data isn't iterable? is there then a reason to use this class? The top line of the doc string seems to contradict this. Also, the description of 'token' seems to imply that data ought to be iterable, although it's not very clear what the 'interval' of a data entry would be.

Reply:
The reason I make it to Any at the first time is that users still can put other types to it without throwing any error, but on the other hand it violate the purpose of this class.  I will change the type hint to Iteralbe.